### PR TITLE
Added WinGet downloads

### PR DESCRIPTION
**Pull request by Mailaender**
https://github.com/microsoft/winget-pkgs/pull/1163

### CODE REVIEW

File: content/download.html [hunk 1]
```diff
@@ -147,6 +150,7 @@ <h3>Community-maintained packages</h3>
       <pre>$ xbps-install openra</pre>
     </dd>
   </dl>
+  <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
```
Comment (pchote) [inline content/download.html:153]: This is duplicating the disclaimer that is already stated above!

Comment (Mailaender) [reply content/download.html:153]: I removed the statement above.

**Comment by pchote** [comment]
Text changes LGTM, although I personally would have put OpenSUSE after arch instead of at the bottom.

The opensuse-fallback.png is the wrong size though, the distro icons should be 24x24px. Ideally the svg should also have its bounding box set to 24px (which will then give the right fallback size when exported).

**Comment by Mailaender** [comment]
Cleaned the .svg and updated the positioning.

File: content/download.html [hunk 2]
```diff
@@ -147,6 +155,7 @@ <h3>Community-maintained packages</h3>
...
```
Comment (pchote) [inline content/download.html:158]: Please change "this" &rarr; "these" and "source" &rarr; "sources". Should be good to merge then.

Comment (Mailaender) [reply content/download.html]: Done.

**Review by pchote — APPROVED** [review]

**Pull request merged**